### PR TITLE
Add Projects Page, Start Calisthenics Page, Start Home Page, Add Images, Modify Style

### PR DESCRIPTION
**Pull request by ike-elue**
The following adds an image folder, adds a target folder, modifies the style, and adds/modifies 3 sites: index, projects, calisthenics. The image folder contains the images used in the sites. The target folder is the result of running the project. The style.css file is modified to better meet the styles of the sites. The index site is modified to have external links to both internal and external sites. The projects site describes cool technical projects I'm working on. The calisthenics site describes current bodyweight moves I'm working towards and the current progression I'm on. As of this request, the only site that is finished is the projects site.

### CODE REVIEW

File: ikenna-portfolio/portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -16,13 +16,15 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+  //const greetings = ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
```
Comment (sliang3) [inline ikenna-portfolio/portfolio/src/main/webapp/script.js]: In general, we should prefer not to check in commented code (here and below).

File: ikenna-portfolio/portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -1,9 +1,22 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 75%;
+}
+
+.image-format {
+    width: 80%;
+}
```
Comment (sliang3) [inline ikenna-portfolio/portfolio/src/main/webapp/style.css:9]: nit: extra spacing

File: ikenna-portfolio/portfolio/src/main/webapp/projects.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
```
Comment (sliang3) [inline ikenna-portfolio/portfolio/src/main/webapp/projects.html:1]: Is this the same content as target/portfolio-1/projects.html?

File: ikenna-portfolio/portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (ike-elue) [reply ikenna-portfolio/portfolio/src/main/webapp/script.js]: done

File: ikenna-portfolio/portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (ike-elue) [reply ikenna-portfolio/portfolio/src/main/webapp/style.css:9]: done

File: ikenna-portfolio/portfolio/src/main/webapp/projects.html (hunk 3 above)
Comment (ike-elue) [reply ikenna-portfolio/portfolio/src/main/webapp/projects.html:1]: I believe so, I can't see anything that shows they're different.

**Review by lupengscu — COMMENTED** [review]
Are those images in ikenna-portfolio/portfolio/src/main/webapp/master_pages snapshots for the actual webpage?

**Comment by ike-elue** [comment]
> Are those images in ikenna-portfolio/portfolio/src/main/webapp/master_pages snapshots for the actual webpage?

Yes

**Review by lupengscu — APPROVED** [review]

**Review by sliang3 — APPROVED** [review]

**Pull request merged**